src/main/java/org/minions/devfund/bruno/html/HtmlValidator.java

```diff
@@ -0,0 +1,46 @@
+package org.minions.devfund.bruno.html;
+
+import java.util.Queue;
+import java.util.Stack;
+
+/**
+ * Represents the validate the information about a single HTML tags.
+ */
+public final class HtmlValidator {
+
+    /**
+     * Constructor private.
+     */
+    private HtmlValidator() {
+        // Default Constructor.
+    }
+
+    /**
+     * Takes as input a Queue of HtmlTags and return a Stack of HtmlTags that verifies the correctness
+     * of the tag structure.
+     *
+     * @param tags queue tags.
+     * @return a Stack.
+     */
+    public static Stack<HtmlTag> isValidHtml(final Queue<HtmlTag> tags) {
+        Stack<HtmlTag> stack = new Stack<>();
+        for (HtmlTag tag : tags) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)